yarn.lock

```diff
@@ -2625,6 +2625,20 @@ ember-moment@^7.5.0:
     ember-getowner-polyfill "^2.0.1"
     ember-macro-helpers "^0.17.0"
 
+ember-native-dom-event-dispatcher@^0.6.4:
```

Comment:
ya, seems out of sync

Reply:
Oops, I probably accidentally ran yarn. I think these apps all use vanilla npm.

Reply:
Scratch that, this repo has yarn. The guides-app does not. Was switching back and forth.